apps/site/src/components/NavBar/NavBar.tsx

```diff
@@ -16,7 +16,7 @@ export default function NavBar() {
 	const activeRoute = usePathname();
 
 	return (
-		<nav className={styles.nav}>
+		<div className={styles.nav}>
```

Comment:
Thought: do we even need this parent element?

Reply:
I originally added the parent element to implement the functionality of the ZotHacks logo moving down with the nav bar drop-down. I put the ZotHacks logo as a sibling of the rest of the nav bar to allow for this

Reply:
I think the sibling behavior is possible with just a fragment, but I suppose the parent styles are necessary if we want sticky behavior, but that doesn't seem to be working right now. Could we reconfirm with graphics what the desired behavior is?